internal/pkg/cli/storage_init.go

```diff
@@ -567,13 +567,12 @@ func BuildStorageInitCmd() *cobra.Command {
 		GlobalOpts: NewGlobalOpts(),
 	}
 	cmd := &cobra.Command{
-		Hidden: true, //TODO remove when ready for production
-		Use:    "init",
-		Short:  "Creates a new storage table in an environment.",
+		Use:   "init",
+		Short: "Creates a new storage resource in an environment.",
```

Comment:
```suggestion
		Short: "Creates a new storage config file in the addons directory.",
```

Reply:
What do you think of this? I just want folks to know that it's not going to **create** a resource.

Reply:
ooh, yes, good point. Way more clear.